src/ansys/geometry/core/designer/body.py

```diff
@@ -358,6 +362,42 @@ def rotate(
         """
         return
 
+    @abstractmethod
+    def scale(
+        self,
+        value: Real,
+    ) -> None:
+        """
+        Scale the geometry body by the given value.
+
+        Parameters
+        ----------
+        value: Real
+            Value to scale the body by.
+        Returns
+        -------
+        None
```

Comment:
If nothing is returned, no need to add the return section

```suggestion
            Value to scale the body by.
```